<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lesson_1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>330</y>
      <width>831</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tsk1_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="tsk2_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="tsk3_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="tsk1_label_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x  +      x +       = 0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk1_a_TB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>18</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk1_b_TB">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>39</y>
      <width>18</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk1_c_TB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>39</y>
      <width>18</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk1_result_TB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="tsk1_label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>178</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>результат:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tsk1_compute">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>вычислить</string>
    </property>
   </widget>
   <widget class="QLabel" name="tsk1_label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="Line" name="tsk2_line_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="tsk2_line_1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tsk2_label_1">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>сторона А</string>
    </property>
   </widget>
   <widget class="QLabel" name="tsk2_label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>сторона Б</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk2_A_TB">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk2_B_TB">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>190</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk2_angle_TB">
    <property name="geometry">
     <rect>
      <x>477</x>
      <y>163</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="tsk2_label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>угол</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tsk2_deg_RB">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>градусы</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="tsk2_deg_RB_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>радианы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tsk2_compute">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>рассчитать сторону В</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tsk2_res_TB">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
